week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/logged-in-user-context.jsx

```diff
@@ -1,31 +1,40 @@
-import React, { useState } from "react";
+import React, {useState} from "react";
 
 /**
  * This is a simplified version of what a Context can be used for.
- * Many pages/components in a bigger application will need to adjust based on the user that is logged in, for example a profile picture/name in the navigation bar or sending the logged in user along with the request to get only data applicable to them.
+ * Many pages/components in a bigger application will need to adjust based on
+ * the user that is logged in, for example a profile picture/name in the
+ * navigation bar or sending the logged-in user along with the request to get
+ * only data applicable to them.
  * As such it is a good idea to have a central place to handle all of this!
  *
- * We have simplified the logging in/out a bit as we do not change anything in the local storage (usually you would save that information there), but I hope you can see how this allows easy adjustments to the logged in information without too much hassle!
+ * We have simplified the logging in/out a bit as we do not change anything in
+ * the local storage (usually you would save that information there), but I hope
+ * you can see how this allows easy adjustments to the logged in information
+ * without too much hassle!
  */
-
 const LoggedInUserContext = React.createContext({
   loggedInUser: null,
   isLoggedIn: false,
   login: () => {
-    // We throw an error as the code should never get here. If we do then sound the alarm bells!
+    // We throw an error as the code should never get here. If we do then sound
+    // the alarm bells!
     throw Error("LoggedInUserContext>>login() is not declared");
   },
   logout: () => {
-    // We throw an error as the code should never get here. If we do then sound the alarm bells!
+    // We throw an error as the code should never get here. If we do then sound
+    // the alarm bells!
     throw Error("LoggedInUserContext>>logout() is not declared");
   },
 });
 
 /**
- * We've put the Provider in the same file as the Context as the Context and Provider are interconnected.
- * The Provider is only needed once, so we make that a named export and have the Context be the default export.
+ * We've put the Provider in the same file as the Context as the Context and
+ * Provider are interconnected.
+ * The Provider is only needed once, so we make that a named export and have
+ * the Context be the default export.
```

Comment:
This comment was formatted a bit clearer before:

```suggestion
 * We've put the Provider in the same file as the Context as the Context and Provider are interconnected.
 * The Provider is only needed once, so we make that a named export and have the Context be the default export.
```

Reply:
I've left the break at 80 chars but I added spacing to improve readability